src/app/tests/suites/certification/Test_TC_ACE_1_1.yaml

```diff
@@ -13,1461 +13,413 @@
-          [1659090456.656374][4334:4339] CHIP:DMG:                                {
-          [1659090456.656430][4334:4339] CHIP:DMG:                                        EndpointId = 0x0,
-          [1659090456.656493][4334:4339] CHIP:DMG:                                        ClusterId = 0x33,
-          [1659090456.656555][4334:4339] CHIP:DMG:                                        CommandId = 0x0,
-          [1659090456.656613][4334:4339] CHIP:DMG:                                },
-          [1659090456.656672][4334:4339] CHIP:DMG:
-          [1659090456.656724][4334:4339] CHIP:DMG:                                StatusIB =
-          [1659090456.656783][4334:4339] CHIP:DMG:                                {
-          [1659090456.656839][4334:4339] CHIP:DMG:                                        status = 0x87 (CONSTRAINT_ERROR),
-          [1659090456.656895][4334:4339] CHIP:DMG:                                },
-          [1659090456.656946][4334:4339] CHIP:DMG:
-          [1659090456.656983][4334:4339] CHIP:DMG:                        },
-          [1659090456.657021][4334:4339] CHIP:DMG:
-          [1659090456.657103][4334:4339] CHIP:DMG:                },
-          [1659090456.657143][4334:4339] CHIP:DMG:
-          [1659090456.657171][4334:4339] CHIP:DMG:        ],
-          [1659090456.657204][4334:4339] CHIP:DMG:
-          [1659090456.657232][4334:4339] CHIP:DMG:        InteractionModelRevision = 1
-          [1659090456.657259][4334:4339] CHIP:DMG: },
-          [1659090456.657346][4334:4339] CHIP:DMG: Received Command Response Status for Endpoint=0 Cluster=0x0000_0033 Command=0x0000_0000 Status=0x87
-          [1659090456.657384][4334:4339] CHIP:TOO: Error: IM Error 0x00000587: General error: 0x87 (CONSTRAINT_ERROR)
-          [1659090456.657425][4334:4339] CHIP:DMG: ICR moving to [AwaitingDe]
-          [1659090456.657488][4334:4339] CHIP:EM: Sending Standalone Ack for MessageCounter:22897794 on exchange 21249i
-          [1659090456.657555][4334:4339] CHIP:IN: Prepared secure message 0xffff8a3839e8 to 0x0000000000000002 (1)  of type 0x10 and protocolId (0, 0) on exchange 21249i with MessageCounter:124109243.
-      disabled: true
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    # NOTE: This won't work with the command line switch for commissioner node id
+    # TODO: get this working with the command line switch
+    - label: "Write ACL giving admin privilege on all EP0"
+      command: "writeAttribute"
+      attribute: "ACL"
+      arguments:
+          value: [
+                  {
+                      FabricIndex: 1,
+                      Privilege: 5, # administer
+                      AuthMode: 2, # case
+                      Subjects: [112233],
```

Comment:
commissionerNodeId?

Reply:
changed in all places - this was up before that change went in, which is why it started like this